ansible/roles/matterbridge/tasks/main.yml

```diff
@@ -5,9 +5,9 @@
     path: "{{ matterbridge_path }}"
     state: directory
 
-- name: "Template matterbridge config from PATH: {{ matterbridge_template_path }}"
+- name: "Template matterbridge config from: {{ matterbridge_t_path }}"
```

Comment:
It's a bit idiosyncratic, but can we have here the full variable name like it was before? (`matterbridge_template_path`)

Nobody is going to remember what `t` stands for

Reply:
And also bellow?

Reply:
In general is a good practice to use shorter variables.
In this specific case I used a shorted variable to reduce the length of the line 80, see the linting log of Molecule:

```
--> Scenario: 'default'

--> Action: 'lint'

--> Executing: set -e

yamllint .

./tasks/main.yml

  8:81      error    line too long (82 > 80 characters)  (line-length)

  21:9      error    wrong indentation: expected 6 but found 8  (indentation)

  32:9      error    wrong indentation: expected 6 but found 8  (indentation)

./handlers/main.yml

  6:14      warning  truthy value should be one of [false, true]  (truthy)

./vars/main.yml

  10:29     error    too many spaces after colon  (colons)

  10:81     error    line too long (85 > 80 characters)  (line-length)

ERROR: Lint failed: Command 'set -e

yamllint .

' returned non-zero exit status 1.: Command 'set -e
```

We can ignore the linter or think of better shorted variables.